gbfs.md

```diff
@@ -576,10 +578,83 @@ Field Name | Required | Type | Defines
 \-&nbsp;`url` | Optional | URL | URL where the customer can learn more about this pricing plan.
 \-&nbsp;`name` | Yes | String | Name of this pricing plan.
 \-&nbsp;`currency` | Yes | String | Currency used to pay the fare. <br /><br /> This pricing is in ISO 4217 code: http://en.wikipedia.org/wiki/ISO_4217 <br />(e.g. `CAD` for Canadian dollars, `EUR` for euros, or `JPY` for Japanese yen.)
-\-&nbsp;`price` | Yes | Non-negative float OR String | Fare price, in the unit specified by currency. If String, must be in decimal monetary value.
+\-&nbsp;`price` | Yes | String | Fare price, in the unit specified by currency. Must be in decimal monetary value.<br /><br />In case of non-rate price, this field is the total price. In case of rate price, this field is the base price that is charged only once per trip (e.g., price for unlocking) in addition to `per_km_pricing` and/or `per_min_pricing`.
 \-&nbsp;`is_taxable` | Yes | Boolean | Will additional tax be added to the base price?<br /><br />`true` - Yes.<br />  `false` - No.  <br /><br />`false` may be used to indicate that tax is not charged or that tax is included in the base price.
 \-&nbsp;`description` | Yes | String | Customer-readable description of the pricing plan. This should include the duration, price, conditions, etc. that the publisher would like users to see.
+\-&nbsp;`per_km_pricing` | Optional | Array | Array of segments when the price is a function of distance travelled, displayed in kilometers.<br /><br />Total price is the addition of `price` and all segments in `per_km_pricing` and `per_min_pricing`. If this array is not provided, there are no variable prices based on distance.
+&emsp;&emsp;\-&nbsp;`start` | Yes | Non-Negative Integer | Number of kilometers that have to elapse before this segment starts applying.
+&emsp;&emsp;\-&nbsp;`rate` | Yes | String | Rate that is charged for each kilometer `interval` after the `start`. Must be in decimal monetary value. Can be a negative number, which indicates that the traveller will receive a discount.
```

Comment:
I disagree with these being strings. These must not be formatted prices (i.e. prices one would show in the UI). These are inputs to a function that needs to be computed by consumer apps and so they must to be parseable as floats; consumer apps can then format the final price in whatever is the user's locale. Same goes for `price`, but as you said, that is unfortunately a breaking change (maybe it should say "Float for any new implementations"?).

Reply:
That is a very good point, thanks for explaining that! I think the best way to do it would be to revert `rate` to floats and then vote on changing `price` to float, but that change to float wouldn't be implemented until v3.0, and then we write a warning that it will be changing and therefore any new implementations are recommended to be floats. I'll update the PR.